include/nbl/video/IGPUImage.h

```diff
@@ -31,6 +31,18 @@ class NBL_API IGPUImage : public asset::IImage, public IDeviceMemoryBacked, publ
 			// stuff below is irrelevant in OpenGL backend
 			E_TILING tiling = ET_OPTIMAL;
 			E_LAYOUT initialLayout = EL_UNDEFINED;
+			
+			SCreationParams() = default;
+
+			SCreationParams(const asset::IImage::SCreationParams& params) 
+				: asset::IImage::SCreationParams(params)
+			{ }
```

Comment:
dont add the constructors, it disables initializer list initialization like `{}`

Reply:
Ok